types/fuzzy-finder/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for fuzzy-finder 1.0
+// Project: https://github.com/tiaanduplessis/fuzzy-finder
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = fuzzyFinder;
```

Comment:
it supports global script, doesn't (UMD)?
https://github.com/tiaanduplessis/fuzzy-finder#install

Reply:
ops, sorry, just notice exports, skip this one